js/config.js

```diff
@@ -35,7 +81,10 @@ var config = {
   popularTweetWeight: 3,
 
   // Maximum number of popular tweets stroed in the database for each trend
-  popularTweetsStored: 60
+  popularTweetsStored: 60,
+
+  // WOEID ID
+  woeid: 23424977
```

Comment:
Should be more descriptive
eg. Yahoo WOEID of location to get trending topics for.